FF1Lib/FF1Rom.cs

```diff
@@ -102,7 +106,19 @@ public static async Task<FF1Rom> CreateAsync(Stream readStream)
 
 			return rom;
 		}
-
+		public class RandomizerDigest
+		{
+			//This JSON gets embeded into the rom, so the property values have been
```

Comment:
Please a space between `//` and `This` like you do elsewhere. Also you misspelled embedded, but I do appreciate the comment.